api.md

```diff
@@ -909,11 +909,17 @@ SELECT detach_tablespaces('conditions');
 ---
 ## distributed_exec() [](distributed_exec)
 
-This function is used on an access node to execute a SQL command
+This procedure is used on an access node to execute a SQL command
 across the data nodes of a distributed database. For instance, one use
 case is to create the roles and permissions needed in a distributed
 database.
 
+The procedure can run distributed commands transactionally, so a command
+is executed either everywhere or nowhere. However, not all SQL commands can run in a 
+transaction. This can be toggled with the argument `transactional`. Note if the execution 
+is not transactional, a failure on one of the data node will require manual dealing with
```

Comment:
`a failure on one of the data node might require manual cleanup` maybe?

Reply:
@pmwkaa It is not necessary cleanup. The user might use different strategy, e.g., manually redoing the command on the failed node.